include/deal.II/grid/tria.h

```diff
@@ -2964,6 +2982,28 @@ class Triangulation : public Subscriptor
   void load (Archive &ar,
              const unsigned int version);
 
+
+  /**
+    * Declare the (coarse) face pairs given in the argument of this function
+    * as periodic. This way it it possible to obtain neighbors across periodic
+    * boundaries.
+    * This function initializes periodic_face_map that stores the active periodic faces.
```

Comment:
I'd say remove this last sentence -- it states something that the user has no relationship with (namely how a private member variable of the class is initialized).